scarb/src/bin/scarb/args.rs

```diff
@@ -345,7 +346,7 @@ pub struct GitRefGroup {
 #[group(multiple = false)]
 pub struct ProfileSpec {
     /// Specify profile to use by name.
-    #[arg(short = 'P', long, env = "SCARB_PROFILE")]
+    #[arg(short = 'P', long)]
```

Comment:
There is a little problem here, we remove information about the environment variable from `--help` :/

<img width="1082" alt="Zrzut ekranu 2023-10-25 o 08 53 56" src="https://github.com/software-mansion/scarb/assets/3450050/d8cd6b17-0936-4fa7-9a58-82a3dedae9ed">